easybuild/easyconfigs/v/Vampir/Vampir-8.1.0-linux-x86_64.eb

```diff
@@ -0,0 +1,32 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Copyright:: Copyright 2013 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+# License::   New BSD
+#
+# This work is based from experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+easyblock='Binary'
+
+name = "Vampir"
+version = "8.1.0"
+versionsuffix = "-linux-x86_64"
+
+homepage = 'http://www.vampir.eu'
+description = """The VAMPIR software tool provides an easy-to-use framework that enables
+ developers to quickly display and analyze arbitrary program behavior at any level of detail.
+ The tool suite implements optimized event analysis algorithms and customizable displays that
+ enable fast and interactive rendering of very complex performance monitoring data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['vampir-%(version)s-standard%(versionsuffix)s-setup.bin']
+
+install_cmd = "./vampir-%(version)s-standard%(versionsuffix)s-setup.bin --silent --instdir=%(installdir)s"
```

Comment:
this has been reviewed some time ago; 
I still recommend the following construct, for compactness/maintainability:
`install_cmd = "./" + sources[0] + " --silent --instdir=%(installdir)s"`
re-testing recommended.